spring-cloud-gateway-core/src/test/java/org/springframework/cloud/gateway/test/ssl/SingleCertSSLTests.java

```diff
@@ -48,7 +48,7 @@
 public class SingleCertSSLTests extends BaseWebClientTests {
 
 	@Before
-	public void setup() {
+	public void setup() throws Exception {
```

Comment:
Remove `throws Exception`.

Reply:
Added it because Keystore Instance creation throws exceptions

Reply:
And this is why I'd rather not have business logic in the properties classes.

Reply:
@spencergibb So, should we move the part which does keystore to keyManagerFactory formation + trusted certs collection in a separate utils class or should we put it in GatewayAutoConfiguration